app/src/main/java/fr/free/nrw/commons/di/FixedDaggerFragment.java

```diff
@@ -0,0 +1,65 @@
+package fr.free.nrw.commons.di;
+
+import android.app.Activity;
+import android.content.Context;
+import android.support.v4.app.Fragment;
+
+import javax.inject.Inject;
+
+import dagger.android.AndroidInjector;
+import dagger.android.DispatchingAndroidInjector;
+import dagger.android.support.HasSupportFragmentInjector;
+
+public abstract class FixedDaggerFragment extends Fragment implements HasSupportFragmentInjector {
```

Comment:
Possible rename - see comments on `FixedDaggerAppCompatActivity`

You call out "AppCompatActivity" in another class name, but this is dealing specifically with _Support_ fragments.  For consistency, perhaps add "Support" to the name - `CommonsDaggerSupportFragment`?

Reply:
(Y)